src/schema/v2/homeView/artworkResolvers.ts

```diff
@@ -0,0 +1,92 @@
+import type { GraphQLFieldResolver } from "graphql"
+import type { HomeViewSectionKey } from "./getSectionsForUser"
+import type { ResolverContext } from "types/graphql"
+import { artworksForUser } from "../artworksForUser"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { paginationResolver } from "../fields/pagination"
+
+/*
+ * Resolvers for home view artwork sections
+ */
+
+// the resolvers
+
+const newWorksForYouResolver: GraphQLFieldResolver<
+  any,
+  ResolverContext
+> = async (parent, args, context, info) => {
+  const finalArgs = {
+    // formerly specified client-side
+    maxWorksPerArtist: 3,
+    includeBackfill: true,
+    first: args.first,
+    version: "C",
+    excludeDislikedArtworks: true,
+    excludeArtworkIds: [],
+
+    ...args,
+  }
+
+  const result = await artworksForUser.resolve!(
+    parent,
+    finalArgs,
+    context,
+    info
+  )
```

Comment:
And once we've assembled the necessary args, we lean on existing resolver logic, rather than rebuilding from scratch. 

`artworksForUser` was instructive in that the resolver encapsulates [quite a bit of logic](https://github.com/artsy/metaphysics/blob/d160f6629aeeecf4089cd652a0c433b87af061be/src/schema/v2/artworksForUser/artworksForUser.ts#L47-L90) — would be bad to try and recreate that!

Do we foresee any risks with this approach, i.e. drift between how we invoke these resolvers here, vs how they were "originally" invoked elsewhere?